.travis.yml

```diff
@@ -2,11 +2,24 @@ language: c
 compiler:
   - clang
   - gcc
+
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libcunit1 libcunit1-doc libcunit1-dev
-before_script:
-  - cmake .
+
 script:
-  - make
-  - make test
+  - mkdir -p ../libuast-build && cd ../libuast-build && cmake ../libuast && make && make test
+
+after_success:
+  - git stash --all
```

Comment:
Now that the build is in a separate dir, this shouldn't be needed anymore, right?